.github/workflows/bdd-e2e.yml

```diff
@@ -0,0 +1,134 @@
+name: Run BDD end to end tests
```

Comment:
Thanks a lot Albert for you work, I will share the ticket with the repository with access to Intel network. This repo will aim to contain all the e2e workflows.

Reply:
Thanks, I've removed the GHA workflows